SDC-seed/PostgreSQL/listingCSVMaker.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+
+const writeListings = fs.createWriteStream('listingTable.csv');
+writeListings.write('listingId,reviews,avatar\n', 'utf8');
```

Comment:
The third column header here is avatar, but below it looks like the third piece of data inserted is price. This might be intentional because I don't know what Will's component is like but want to make a note and double check with you.